src/react/ColorPicker/components/SaturationSpectrum.tsx

```diff
@@ -57,24 +57,24 @@ export default function SaturationSpectrum(props: SaturationSpectrumProps) {
         }
     }
 
-    function initContainerEl() {
```

Comment:
Should not remove this function, instead rename and use it later in handleMouseUp, handleMouseMove... to avoid repetition.